Norm_end_to_end.Rmd

```diff
@@ -395,35 +451,71 @@ intern_standard$CV_NOMIS <- rowRsd(
 
 cpt <- paste("Internal standards with detected and assigned features coefficient
              of variation before and after different method of normalization.")
-pandoc.table(intern_standard[, c("abbreviation", "CV_raw", "CV_MRM", 
-                                 "CV_MRM_500", "CV_ruvr", 
+pandoc.table(intern_standard[, c("abbreviation", "CV_raw", "CV_MRM",
+                                 "CV_MRM_500", "CV_ruvr",
                                  "CV_ruv3", "CV_NOMIS")],
              style = "rmarkdown", caption = cpt)
 ```
+
 At first glance it seems that MRM 500 works better than MRM. and RUV random
-completely remove variation.. .
+completely remove variation.. . RUV and NOMIS used however the internal
+standards to estimate noise (correct?), thus, it would be expected that CV
+of internal standards decreases considerably for these methods.
+
+Estimating the CV for internal standards only in study samples - to avoid any
+influence from QC samples.
+
+```{r include=TRUE, results = "asis"}
+index_study <- which(res$phenotype != "QC")
+
+is_study <- data.frame(
+    CV_raw = rowRsd(assay(res, "raw_filled")[intern_standard$feature_id,
+                                              index_study], na.rm = TRUE),
+    CV_MRM = rowRsd(assay(res, "median_filled")[intern_standard$feature_id,
+                                                 index_study], na.rm = TRUE),
+    CV_MRM_500 = rowRsd(
+        assay(res, "median_filled_500")[intern_standard$feature_id,
+                                        index_study], na.rm = TRUE),
+    CV_ruvr = rowRsd(
+        assay(res, "ruvr")[intern_standard$feature_id,
+                           index_study], na.rm = TRUE),
+    CV_ruv3 = rowRsd(
+        assay(res, "ruv3")[intern_standard$feature_id,
+                           index_study], na.rm = TRUE),
+    CV_NOMIS = rowRsd(
+        assay(res, "NOMIS")[intern_standard$feature_id,
+                            index_study], na.rm = TRUE)
+)
+cpt <- paste("Internal standards with detected and assigned features coefficient
+             of variation before and after different method of normalization.")
+is_study$abbreviation <- intern_standard$abbreviation
+pandoc.table(is_study[, c("abbreviation", "CV_raw", "CV_MRM",
+                          "CV_MRM_500", "CV_ruvr",
+                          "CV_ruv3", "CV_NOMIS")],
+             style = "rmarkdown", caption = cpt)
+```
 
 Study samples
 coeff variation of study samples should not go down too much
 
-```{r include=FALSE}
+```{r include=TRUE, results = "asis"}
 index_study <- res$phenotype %in% c("CTR", "CVD")
 
-study_sample <- cbind(CV_Raw = rowRsd(assay(res, "raw")[, index_study],
-                                      na.rm = TRUE),
-            CV_MRM = rowRsd(
-    assay(res, "median_filled")[, index_study], na.rm = TRUE),
+study_sample <- cbind(
+    CV_Raw = rowRsd(assay(res, "raw")[, index_study], na.rm = TRUE),
```

Comment:
Ah, no, also here we should use `"raw_filled"` instead of `"raw"`. Forgot to fix.

Reply:
Ok ! i will change it after we merge